src/Service.php

```diff
@@ -82,8 +82,14 @@ private function fetchFlags(): void
 
     public function getFlag(string $key): Flag
     {
+        $envKey = 'FEATURE_FLAG_' . $key;
```

Comment:
I would have `FEATURE_FLAG_` as a FF_ENV_PREFIX constant in the class

Reply:
done